core/snmp/impl-snmp4j/src/main/java/org/opennms/netmgt/snmp/snmp4j/Snmp4JTrapNotifier.java

```diff
@@ -113,6 +119,10 @@ protected VariableBinding getVarBindAt(int i) {
         protected void processVarBindAt(int i) {
             SnmpObjId name = SnmpObjId.get(getVarBindAt(i).getOid().getValue());
             SnmpValue value = new Snmp4JValue(getVarBindAt(i).getVariable());
+            if(getVarBindAt(i).getVariable().toString().matches("[a-zA-Z:]*"))
```

Comment:
Please remove this statement. The values should not be changed before they are sent to the processVarBind() method.